modules/microshift-install-multus-rpm.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-multus_{context}"]
+== Installing the multiple networks plugin
+
+The {microshift-short} Multus Container Network Interface (CNI) plugin is not installed by default. If you want to attach additional networks to a pod for high-performance network configurations, install the `microshift-multus` RPM package. You can install the plugin for multiple networks during either a new installation or a running cluster. Use this procedure to install the {microshift-short} Multus CNI plugin alongside a new {microshift-short} installation.
+
+[IMPORTANT]
+====
+The Multus RPM cannot be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-networking-multus
+----
+
+.Next steps
+. Continue with your {microshift-short} installation, including any add-ons.
+
+. After you have finished installing all of the RPMs you want to include, start the {microshift-short} service. The {microshift-short} Multus CNI plugin is automatically deployed.
```

Comment:
- if `microshift-multus` was installed together with `microshift` RPM, then it should just work (i.e. `dnf install microshift microshift-multus`) - this wouldn't work for RPM upgrade though, only first install
- if `m-multus` was installed later (**to an existing cluster**), then CRIO needs to be restarted to notice it and start using as the CNI, however, we want all the Pods to be created with Multus (even if they don't have additional interfaces/NADs), so here we recommend doing a system reboot: https://github.com/openshift/enhancements/blob/master/enhancements/microshift/multus-cni-for-microshift.md#adding-to-existing-microshift-cluster